app/web/pages/group/[id]/[slug].tsx

```diff
@@ -0,0 +1,35 @@
+import { appGetLayout } from "components/AppRoute";
+import GroupPageComponent from "features/communities/GroupPage";
+import NotFoundPage from "features/NotFoundPage";
+import { GetStaticPaths, GetStaticProps } from "next";
+import { useRouter } from "next/router";
+import nextI18nextConfig from "next-i18next.config";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import stringOrFirstString from "utils/stringOrFirstString";
+
+export const getStaticPaths: GetStaticPaths = () => ({
+  paths: [],
+  fallback: "blocking",
+});
+
+export const getStaticProps: GetStaticProps = async ({ locale }) => ({
+  props: {
+    ...(await serverSideTranslations(
+      locale ?? "en",
+      ["global", "communities"],
+      nextI18nextConfig
+    )),
+  },
+});
+
+export default function GroupPage() {
```

Comment:
We should probably hide this behind the `is_communities_part2_enabled` feature flag as before and return 404 as these pages aren't really ready yet? Especially now this is gonna be server rendered, not sure we want the search engine crawlers to be indexing broken pages 😅